observability/opentelemetry/src/main/resources/schema/wildfly-opentelemetry_1_0.xsd

```diff
@@ -21,15 +21,14 @@
                         </xs:documentation>
```

Comment:
You need to create a new schema (and parser.)

Reply:
I remembered about that part after I left the house. On it now.